src/pybind/mgr/cephadm/templates/services/ingress/haproxy.cfg.j2

```diff
@@ -74,7 +74,7 @@ backend backend
     balance static-rr
     option httpchk HEAD / HTTP/1.0
     {% for server in servers %}
-    server {{ server.name }} {{ server.ip }}:{{ server.port }} check weight 100
+    server {{ server.name }} {{ server.ip }}:{{ server.port }} check weight 100 inter health_check_interval
```

Comment:
I'd made a comment before

------
```suggestion
    server {{ server.name }} {{ server.ip }}:{{ server.port }} check weight 100 inter {{ health_check_interval }}
```
This needs the `{{ }}` otherwise it will just print the string "health_check_interval" into the conf instead of "2s" or whatever the variable is set to. That should also fix the unit test failures that are causing make check to not pass.

---
and it was marked resolved, but the issue is still present. This is the reason for the make check failure and means the config always just ends up with
`weight 100 inter health_check_interval` instead of something like `weight 100 inter 2s`